python/grass/temporal/temporal_raster_base_algebra.py

```diff
@@ -1208,10 +1208,10 @@ def p_arith2_operation(self, t):
                         break
                     if count == 0:
                         # Set map name.
-                        name = map_new.get_id()
+                        map_new.get_id()
                     else:
                         # Generate an intermediate map
-                        name = self.generate_map_name()
+                        self.generate_map_name()
```

Comment:
This looks like it could be removed, but `generate_map_name` actually modifies a dictionary, so I think we need to keep this as you have it.